website/sidebars.json

```diff
@@ -27,8 +28,16 @@
     "Advanced": [
       "extension/overview",
       "extension/verification-methods",
-      "extension/identity-proofs"
+      "extension/identity-proofs",
+      "extension/custom-renderer",
+      "extension/configuring-dns"
     ],
```

Comment:
wanna either stick to `Advanced` + `advanced/overview` etc or `Extension`+ `extension/overview` etc?

Reply:
I will change to advanced/xxx

Reply:
changed